tap_exacttarget/__init__.py

```diff
@@ -218,10 +150,9 @@ def main():
             do_sync(args)
         else:
             LOGGER.info("No properties were selected")
-    except RuntimeError as exception:
-        LOGGER.error(str(exception))
-        LOGGER.fatal("Run failed.")
-        exit(1)
+    except Exception as exception:
```

Comment:
FYI, we added a [decorator](https://github.com/singer-io/singer-python/blob/459a90e9b067538d0645cba3177c71e670bddaf2/singer/utils.py#L174) in singer utils that you can add to main() --  it just wraps a function in a try/catch with critical logging.  It would require updating singer-python for this tap, which would mean more testing -- so probably not worth it since, functionally, it doesn't change anything.

Reply:
No time like the present. :)